cucumber/authenticators_azure/features/step_definitions/authn_azure_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^I successfully set Azure variables$/) do
+  set_azure_provider_uri_variable
```

Comment:
so if I am understanding here, you are either using the default value for `set_azure_provider_uri_variable` which is `azure_provider_uri` or setting it. If you use `azure_provider_uri` then you are using the `AZURE_TENANT_ID`. What did we gain from this? The ability to take from the env or be dynamic?

Reply:
In future tests we will give custom inputs for the provider uri, such that will end with 502 & 504 response codes.